vips/vip-383/bscmainnet.ts

```diff
@@ -0,0 +1,61 @@
+import { ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+import { FAST_TRACK_TIMELOCK } from "src/vip-framework";
+
+const vip383 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-383: [Ethereum] Prime Adjustment Proposal - Q4 2024",
```

Comment:
I think we don't use colums in VIP titles usually, the only occurrence is VIP-381
![Uploading image.png…]()


```suggestion
    title: "VIP-383 [Ethereum] Prime Adjustment Proposal - Q4 2024",
```

Reply:
https://github.com/VenusProtocol/vips/pull/408/commits/80094dc9ffc2f97103f560564c5bdc88f4626c55